client/blocks/reader-join-conversation/style.scss

```diff
@@ -0,0 +1,69 @@
+.reader-join-conversation-dialog {
+	border-radius: 6px; /* stylelint-disable scales/radii */
+	width: 321px;
+
+	.dialog__action-buttons-close {
+		padding: 24px;
+	}
+
+	.dialog__content {
+		padding: 24px 32px;
+	}
+}
+
+.reader-join-conversation-dialog__content {
+	width: 257px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+
+	h1 {
+		color: var(--color-neutral-70);
+		font-size: 1.25rem;
+		font-weight: 600;
+		margin-bottom: 9px;
+	}
+
+	p {
+		font-size: 1rem;
+		text-align: center;
+	}
+
+	svg.wordpress-logo {
+		fill: var(--color-neutral-70);
+	}
+
+	button.components-button.is-primary {
+		background-color: #117ac9;
+		border-color: #117ac9;
+
+		&:active:not(:disabled),
+		&:hover:not(:disabled),
+		&:focus:not(:disabled) {
+			background-color: #0e64a5;
+			border-color: #0e64a5;
+		}
+	}
+
+	button.components-button.is-link {
+		padding: 20px 0 6px 0;
+		color: #1d2333;
+
+		&:hover:not(:disabled) {
+			color: var(--studio-blue-50);
+		}
+	}
+
+
+}
+
+@media only screen and (max-width: 600px) {
+	.reader-join-conversation-dialog__content {
+		width: 100%;
+	}
+}
+
+.dialog__backdrop,
+.dialog__backdrop.card {
+	background-color: rgba(var(--color-neutral-70-rgb), 0.8) !important;
+}
```

Comment:
@eoigal  I know I'm very late to the party, but I'm curious if we really need the `!important` here? It looks like this takes over and makes some models within Jetpack Cloud look weird:

![image](https://github.com/Automattic/wp-calypso/assets/1749918/35256d16-cdb5-490d-95f8-0cf5ebd64801)

The background here should be white, as you can see in this PR: https://github.com/Automattic/wp-calypso/pull/79135